zulipterminal/config.py

```diff
@@ -1,18 +1,42 @@
 KEY_BINDINGS = {
     'k':
-        ('up', 'scroll up'),
+        ('up', 'Previous message'),
     'j':
-        ('down', 'scroll down'),
+        ('down', 'Next message'),
     'h':
         ('left', 'Go left'),
     'l':
         ('right', 'Go right'),
     'K':
-        ('page up', 'scroll to top'),
+        ('page up', 'Scroll to top'),
     'J':
-        ('page down', 'scroll to bottom'),
+        ('page down', 'Scroll to bottom'),
     'G':
         ('end', 'Go to last message in view'),
+    'r':
```

Comment:
It would be worth changing the structure of `KEY_BINDINGS`. 

Something like 
```python
...
'NEXT_MESSAGE': {'keys': {'j', 'down'}, 'help_text': 'Next message'},
'REPLY': {'keys': {'r'}, 'help_text': 'Reply to a message'},
...
```
might be good. 

This would allow us to replace all the hard-coded checks for keys (like `key == "r"` to something like `key in KEY_BINDINGS['REPLY']['keys']`).  We could also add a helper class to add methods for these kind of checks, etc. 

Later, this would allow for making it easy for users to overwrite/add more keys for specific actions.